css/style.css

```diff
@@ -845,17 +854,11 @@ button:hover {
 
 @media (max-width: 480px) {
 	/* hide board actions on mobile */
-	#board-actions {
+	#board-header-controls {
 		display: none !important;
```

Comment:
I agree 100%, @juliushaertl

@artemanufrij yeah, something like that. But maybe we should stack them vertically, because it seems to be the convention. Archive and Details buttons should then have title visible